### PR TITLE
Integration Utilities and Testing Env

### PR DESCRIPTION
**Pull request by mustermeiszer**
A lot of utilities we are going to need and testing if the provided env works.

### CODE REVIEW

File: runtime/integration-tests/src/pools/utils/accounts.rs [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+pub enum Keyring {
+	Admin,
+	TrancheInvestor(u32),
+	Alice,
+	Bob,
+	Charlie,
+	Dave,
+	Eve,
+	Ferdie,
+	Costume(&'static str),
```
Comment (offerijns) [inline runtime/integration-tests/src/pools/utils/accounts.rs:33]: What does this refer to? Should it be `Custom`?

**Review by offerijns — APPROVED** [review]
One question, otherwise LGTM

File: runtime/integration-tests/src/pools/utils/accounts.rs (hunk 1 above)
Comment (mustermeiszer) [reply runtime/integration-tests/src/pools/utils/accounts.rs:33]: ^^ Yes. I will change that.

Comment (mustermeiszer) [reply runtime/integration-tests/src/pools/utils/accounts.rs:33]: I wil merge it an rename it different PR as otherwise the CI runs again...

**Pull request merged**